lib/inspec/rspec_json_formatter.rb

```diff
@@ -748,13 +748,73 @@ def status_type(example)
   end
 end
 
-class InspecRspecJUnit < RSpecJUnitFormatter
+class InspecRspecJUnit < InspecRspecJson
```

Comment:
It is so good to see that this is now based on our InSpec JSON generator. Awesome @jkerry